src/pages/registration/components/RegistrationNotFound.tsx

```diff
@@ -0,0 +1,39 @@
+import { Container, Heading, Flex, Text, Divider } from '@chakra-ui/layout';
+import { Button, Box } from '@chakra-ui/react';
+import { HiExternalLink } from 'react-icons/hi';
+import { useParams } from 'react-router';
+
+import { useDarkMode } from 'lib/hooks/useDarkMode';
+import { getReadableTerm } from 'lib/utils/terms';
+
+export function RegistrationNotFound() {
```

Comment:
My idea for this is it should be a general component that can be used on the Registration page and Booklist page since they have the same message and we want it to be easy on both pages to go to their timetable
I would rename this `CoursesNotFound` and move it into our `common` folder